src/components/pages/devices/flyouts/SIMManagement/SIMManagement.js

```diff
@@ -76,6 +86,10 @@ export class SIMManagement extends LinkedComponent {
                 </Section.Content>
               </Section.Container>
             }
+            <BtnToolbar>
+              {!isPending && <Btn primary={true} disabled={!provider} onClick={this.showProvider} type="submit">{t('devices.flyouts.new.apply')}</Btn>}
+              <Btn svg={svgs.cancelX} onClick={onClose}>{isPending ? t('devices.flyouts.new.close') : t('devices.flyouts.new.cancel')}</Btn>
+            </BtnToolbar>
```

Comment:
@mechaffin I had to resolve conflicts. Can you make sure this is still accurate, with `Protected` section?

Reply:
Yes, this looks correct for the Protected section.